### ISSUE TITLE
App requires multiple login hits

### ISSUE THREAD

**Issue by adikakade31** [codepath-android-fall16-group-8/travel#19]
After first login with username/password, the app requires multiple hits to login button to take to home screen. Is it because device doesn't have Facebook App?
Link to video walkthrough for #4 
https://www.dropbox.com/s/kl00o85n0zczpl0/device-2016-11-12-114038.mp4?dl=0

**Comment by prav90** [comment]
@adikakade31 Don't know what's going on behind, will check it out today

**Comment by adikakade31** [comment]
Yup, me too! Getting this error every time it asks to login again. 
E/MultiWindowProxy: getServiceInstance failed!

**Comment by heyhuyen** [comment]
Yeah, this problem goes away if you are logged into the facebook app on the device. That was the case for me.

There is a way to add a custom chrome tab so if the user is logged in to facebook (via chrome browser?) that they won't get asked to login again and again. I had tried to start implementing it but for some reason the CustomTabActivity could not be found. Instructions here: https://developers.facebook.com/docs/facebook-login/android#10--enable-chrome-custom-tabs
